### PR TITLE
UP-4805: update jquery to version 1.11.0 and backbone to version 1.3.3

### PR DESCRIPTION
**Pull request by Josh31415**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Merging note, this PR depends on https://github.com/Jasig/resource-server/pull/54

File: uportal-war/src/main/webapp/WEB-INF/jsp/DynamicRespondrSkin/skinConfig.jsp [hunk 1]
```diff
@@ -106,7 +106,7 @@
     </div>
 </div>
 
-<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.10.2/jquery-1.10.2.min.js"/>"></script>
+<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>"></script>
```
Comment (drewwills) [inline uportal-war/src/main/webapp/WEB-INF/jsp/DynamicRespondrSkin/skinConfig.jsp]: I'm not confident the `skinConfig` view should have it's own copy of jQuery.  I will look into it and take care of it as I process this PR.

File: uportal-war/src/main/webapp/WEB-INF/jsp/Search/search.jsp [hunk 2]
```diff
@@ -134,7 +134,7 @@
     </div>
 </div>
 
-<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.10.2/jquery-1.10.2.min.js"/>"></script>
+<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>"></script>
```
Comment (drewwills) [inline uportal-war/src/main/webapp/WEB-INF/jsp/Search/search.jsp]: I'm not confident the `search` view should have it's own copy of jQuery.  I will look into it and take care of it as I process this PR.

File: uportal-war/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp [hunk 3]
```diff
@@ -50,7 +50,7 @@
     </div>
 </div>
 
-<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.10.2/jquery-1.10.2.min.js"/>"></script>
+<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>"></script>
```
Comment (drewwills) [inline uportal-war/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp]: I'm not confident the `searchLauncher` view should have it's own copy of jQuery.  I will look into it and take care of it as I process this PR.

File: uportal-war/src/main/webapp/media/skins/muniversality/common/common_skin.xml [hunk 4]
```diff
@@ -54,8 +54,8 @@
     <js included="plain" resource="true">/rs/underscore/1.5.2/underscore-1.5.2.js</js>
     <js included="aggregated" resource="true">/rs/underscore/1.5.2/underscore-1.5.2.min.js</js>
 
-    <js included="plain" resource="true">/rs/backbone/1.0.0/backbone-1.0.0.js</js>
-    <js included="aggregated" resource="true">/rs/backbone/1.0.0/backbone-1.0.0.min.js</js>
+    <js included="plain" resource="true">/rs/backbone/1.3.3/backbone-1.3.3.js</js>
+    <js included="aggregated" resource="true">/rs/backbone/1.3.3/backbone-1.3.3.min.js</js>
```
Comment (drewwills) [inline uportal-war/src/main/webapp/media/skins/muniversality/common/common_skin.xml]: Requires a new build of `resource-server-content` and must be accompanied with a version increment in the root `pom.xml` file.  I will take care of it.

**Comment by drewwills** [comment]
Folks -- I know I indicated that "I will take care of it" on a number of items.  I thought it would be done by now... I had some difficulty with a `resource-server` build.  (But that's done now.)  I should be able tom finish this PR soon.

**Comment by drewwills** [comment]
@Josh31415 -- This one is fully merged.  I made the changes I wrote about, and I applied your changes by-hand because I couldn't `$ git cherry-pick` them.  (This PR apparently comes from an "unknown repository.")

**Comment by drewwills** [comment]
@Josh31415, @ChristianMurphy, and @asgrant -- I applied this change, but it introduced regressions.  :(

Apparently Backbone 1.3.3 requires Underscore 1.8.3 (https://github.com/jashkenas/backbone/issues/4013).  I quickly put underscore 1.8.3 indo my local, and it's much better.  We need to get this update in master.

**Comment by ChristianMurphy** [comment]
Resource Server is currently behind  on the underscore version.
[Underscore 1.6.0 is the latest that is availiable](https://github.com/Jasig/resource-server/tree/master/resource-server-content/src/main/webapp/rs/underscore).
A newer version will need to be added there and a new release cut before the issue can be resolved in uPortal.

**Comment by ChristianMurphy** [comment]
https://github.com/Jasig/resource-server/pull/57

**Pull request closed without merging**